src/modules/account/components/overview/index.tsx

```diff
@@ -11,15 +14,17 @@ type OverviewProps = {
 }
 
 const Overview = ({ customer, orders }: OverviewProps) => {
+  const t = useSafeTranslations()
+
   return (
     <div data-testid="overview-page-wrapper">
       <div className="hidden small:block">
         <div className="text-xl-semi flex justify-between items-center mb-4">
           <span data-testid="welcome-message" data-value={customer?.first_name}>
-            Hello {customer?.first_name}
+            {t(k.HELLO_CUSTOMER)} {customer?.first_name}
```

Comment:
I'd recommend to delegate interpolation of values into strings like `Hello {firstName}`. That gives more flexibility to translators where variables should be used.